### PR TITLE
Scaletesting: update 200k blank repo info

### PR DESCRIPTION
**Pull request by sanderginn**

### CODE REVIEW

File: content/departments/engineering/dev/tools/scaletesting.md [hunk 1]
```diff
@@ -273,9 +273,16 @@ sg client codehost add-github \
 See https://ghe.sgdev.org/testing which is a replica of https://github.com/londonappbrewery on our GitHub instance.
 They are owned by the `testing` user, who can write on those repos.
 
-#### Organization with 200k blank repos (single initial commit with `README.md`)
+#### 200k blank repos (single initial commit with `README.md`)
```
Comment (burmudar) [inline content/departments/engineering/dev/tools/scaletesting.md]: I think the part in brackets can be a bulletpoint rather

**Review by burmudar — APPROVED** [review]

File: content/departments/engineering/dev/tools/scaletesting.md [hunk 2]
```diff
@@ -273,9 +273,16 @@ sg client codehost add-github \
...
+
+- 198k repositories assigned to `main-org`: https://ghe-scaletesting.sgdev.org/main-org
+  - 4750 teams have permissions on 27 or 28 repositories
```
Comment (Piszmog) [inline content/departments/engineering/dev/tools/scaletesting.md]: Can you provide the link to the teams or how someone would determine what teams these are?

File: content/departments/engineering/dev/tools/scaletesting.md [hunk 3]
```diff
@@ -273,9 +273,16 @@ sg client codehost add-github \
...
+  - 200 teams have permissions on 78 repositories
+  - 50 teams have permissions on 985 repositories
+  - 1000 repositories each have 3 users assigned as collaborators who are the only users with access
```
Comment (Piszmog) [inline content/departments/engineering/dev/tools/scaletesting.md]: Same with these repos. How would someone know what these are

**Review by marekweb — APPROVED** [review]

File: content/departments/engineering/dev/tools/scaletesting.md (hunk 2 above)
Comment (sanderginn) [reply content/departments/engineering/dev/tools/scaletesting.md]: Took a while but addressed your points. Thanks!

**Review by kalanchan — APPROVED** [review]
holiday commits :letsgo: !!

**Pull request merged**